src/Tracy/Helpers.php

```diff
@@ -157,6 +157,43 @@ public static function errorTypeToString(int $type): string
 	}
 
 
+	/**
+	 * Truncates UTF-8 string to maximal length.
+	 */
+	public static function truncate(string $s, int $maxLen = 32768, string $append = "\u{2026}"): string
```

Comment:
This is too complex for Tracy. You should extract the truncate code from Dumper

Reply:
https://github.com/nette/tracy/blob/44349212f8856cf1cacb0ab2b62c2b30e4be6f93/src/Tracy/Dumper/Dumper.php#L519-L531

Reply:
I added method Dumper::truncateString()

Reply:
@dg Thank you and replaced `Helpers::truncate` to `Dumper::encodeString`.